docker-compose.yml

```diff
@@ -362,3 +362,32 @@ services:
     volumes:
       - ./api/app/tests/coverage:/var/www/tests/coverage
       - ./build/coverage-api:/var/www/build/coverage-api
+
+  end-to-end-tests:
```

Comment:
The only thing I'm foxed by is why this has been added, as it seems to have nothing to do with the other JS stuff in the PR. Is this intentional?

Reply:
Thanks. You're right it's unrelated. I started doing the end-to-end tests first before running into the issues with js commands in the Makefile and missed this when backing out changes related to that